lib/validate.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
       compress: [boolean().allow(null)],
       headers: object().allow(null),
       historyFallback: [boolean(), object()],
-      hmr: boolean(),
+      hmr: [boolean(), 'refresh-on-failure'],
```

Comment:
Based on the CI error (`An option was passed to WebpackPluginServe that is not valid`), this may we wrong or alternatively there's some logic error.

Reply:
Should be `union([boolean(), literal('refresh-on-failure')])` with superstuct.

Reply:
Cool, thanks. I'll update.